ikasaneip/visualisation/dashboard/src/main/java/org/ikasan/dashboard/schedule/DashboardJob.java

```diff
@@ -0,0 +1,20 @@
+package org.ikasan.dashboard.schedule;
+
+import org.quartz.Job;
+
+public interface DashboardJob extends Job {
```

Comment:
should this interface have  some default implementation.

Should BusinessStreamNotificationJob be extending default imple of  DashboardJob

Reply:
i am happy to leave this as an interface that all dashboard scheduled tasks implement.